src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -158,7 +158,7 @@ The following five properties must be configured:
 
 TIP: There are useful templates for different databases in *PROACTVE_HOME/config/scheduler/templates/* and *PROACTVE_HOME/config/rm/templates/*.
 
-The *job-planner*, *catalog*, *scheduling-api*, and *proactive-cloud-watch* microservices must contain a database configuration.
+The *job-planner*, *catalog*, *scheduling-api*, *cloud-automation-service* and *proactive-cloud-watch* microservices must contain a database configuration.
```

Comment:
Maybe my question is stupid, but why not just *cloud-automation*? If so, shouldn't we add service to all other services?

Reply:
Your question is not stupid. We have this inconsistency indeed. And eventually, we can change its name. But now it is called `cloud-automation-service` (at least at war level).

Reply:
Fair enough. I think it should be changed. Maybe in the futur if it takes time. Thanks!